apps/designer/app/designer/features/style-panel/shared/css-value-input/css-value-input.tsx

```diff
@@ -251,6 +255,21 @@ export const CssValueInput = ({
       return;
     }
 
+    let styleInput = parseCssValue(property, value.value);
+
+    if (styleInput.type !== "invalid") {
+      props.onChange(styleInput);
+      return;
+    }
+
+    const unit = "unit" in value ? value.unit ?? "px" : "px";
+    styleInput = parseCssValue(property, `${value.value}${unit}`);
```

Comment:
Strange logic:
1. In the second `parseCssValue` we use `value.unit` if present, which makes me think that `value.unit` is potentially important
2. But in the first `parseCssValue` we ignore `value.unit`

Reply:
See comments `// Probably value is already valid, use it` 

My idea is that user raw input has max priority, if it's valid, let's use it.
Then I just try to get what the user probably wanted if the raw input is invalid.

Reply:
After discussion, line-height will not work with this logic. Will write test for `value` conversion